README.md

```diff
@@ -68,6 +68,10 @@ spec:
         cool-label: foo
       annotations:
         cool-annotation: bar
+    securityContext:
+      runAsUser: 1000
+      runAsGroup: 1000
+      runAsNonRoot: true
```

Comment:
i think you have reason and both you be done, my bad :) i missed a part.

I think i only needed this part to make it working properly through helm and our high constrainted environment, like you said USER is define on the other and kube is able to manage it properly without modification (long time since i did tests on it)

Reply:
just re-reading carefully the code, in the operator i modified the common Pod structure which means both can be set if you want, maybe i should edit the README again to show that :)

Reply:
Yes, this way is much clearer, thanks! The unit tests are failing now though, please see: https://github.com/grafana/k6-operator/runs/4564203207

TBH, we'll be adding new stages / containers to k6-operator in the near future (there're open PRs for that right now) so we might need to re-visit this logic later again: i.e. it might not make sense to add security context separately for each stage. I suppose that some people would be fine with default (not setting security context) but some might want to have one, global security context for everything test-related :thinking: But that's something to think on in the future. I think we should be fine with your approach for now :slightly_smiling_face: Please check the unit tests and I'll merge it on the next week!